script.module.requests/addon.xml

```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.requests" name="requests" version="2.25.1+matrix.1" provider-name="Python Software Foundation">
+<addon id="script.module.requests" name="requests" version="2.27.0+matrix.1" provider-name="Kenneth Reitz">
```

Comment:
The requests project was transferred to PSF 2 years ago

Reply:
I am happy to change that but both the PKG-INFO as well as PyPi lists Kenneth Reitz as the author. I was under the impression that the original author/maintainer was to be used for *provider-name* . Do you want me to change this back to PSF?

Reply:
Updated version to 2.27.1

Reply:
@L2501 , how do you want to proceed regarding my last question?

Reply:
@enen92

Reply:
Keep it as PSF if possible